app/v2/notifications/post_notifications.py

```diff
@@ -249,6 +253,33 @@ def post_notification(notification_type):
     return jsonify(resp), 201
 
 
+def triage_notification_to_queues(notification_type, signed_notification_data, template):
+    """Determine which queue to use based on notification_type and process_type
+
+    Args:
+        notification_type: Type of notification being sent; either SMS_TYPE or EMAIL_TYPE
```

Comment:
by convention it would be something like this

```
notification_type: str
Type of notification sent .....
```

The docstring will indicate the types or you can use mypy type.